Assets/Src/AuthController/SampleAuth.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Src.AuthController.TokenProviders.TokenProvidersFactory;
+using Src.Caching;
+using UnityEngine;
+
+namespace Src.AuthController
+{
+    public class SampleAuth : MonoBehaviour, IAuthView
```

Comment:
Use manual tests only to test the program aesthetics from user's view or to test code usability.

Reply:
Also, all test related code should be in Tests folder, not in Src